application/helpers/admin/export/Writer.php

```diff
@@ -321,6 +321,7 @@ protected function transformResponseValue($value, $fieldType, FormattingOptions
                 $value = $response[$column];
                 if (isset($oSurvey->fieldMap[$column]) && $oSurvey->fieldMap[$column]['type']!='answer_time' && $oSurvey->fieldMap[$column]['type']!='page_time' && $oSurvey->fieldMap[$column]['type']!='interview_time')
                 {
+                    $value = getExtendedAnswer($oSurvey->id, $column, $value, $sLanguageCode);
```

Comment:
I don't think it's OK to return getExtendedAnswer for 'short' answers. And i'm sure we need another function. getExtendedAnswer do "Long answer [answer code]" for single choice